### ISSUE TITLE
Hello, may I ask if ATST Frame can publicly disclose the models and scripts used for inference?

### ISSUE THREAD

**Issue by Angelalilyer** [Audio-WestlakeU/audiossl#12]
I only found the training code, and my test dataset is unlabeled. I would like to try using ATST Frame to detect sound events in a new dataset. Perhaps you have inference code and models? Looking forward to your reply!

**Comment by lmaxwell** [comment]
Hi,
Currently, only self-supervised pretrained checkpoint are pulic in this repo. But we are happy to make the finetuned checkpoint public.
As you are targeting for detercting sound events,  do you mean the checkpoint and inference code of  ATST-Frame finetuned on strongly-labeled audioset?

**Comment by Angelalilyer** [comment]
Thanks for your reply~！
Yes, that is exactly what i need!  So I wonder When you will release the “checkpoint and reference code of ATST Frame refined on strongly labeled audioset” ?

**Comment by SaoYear** [comment]
Hi, I would let you know once the uploading finished ; )

**Comment by SaoYear** [comment]
@Angelalilyer You could try this [checkpoint file](https://drive.google.com/file/d/1RWjnEnkDb-7fmNlqIcM-MovTnddOzLsh/view?usp=sharing), hope it helps!

**Comment by Angelalilyer** [comment]
> @Angelalilyer You could try this [checkpoint file](https://drive.google.com/file/d/1RWjnEnkDb-7fmNlqIcM-MovTnddOzLsh/view?usp=sharing), hope it helps!

Thank you so much!!

**Comment by Angelalilyer** [comment]
> @Angelalilyer You could try this [checkpoint file](https://drive.google.com/file/d/1RWjnEnkDb-7fmNlqIcM-MovTnddOzLsh/view?usp=sharing), hope it helps!

Is there a complete inference code available? I tried to modify "audiossl/audiossl/methods/atstframe/downstream/train_strong. py" but kept reporting errors. 
Sorry to bother you again～

**Comment by lmaxwell** [comment]
Did you solve the problem?

**Comment by Angelalilyer** [comment]
> Did you solve the problem?

I tried to write inference code myself, but I couldn't output predicted labels，may I ask if there is a relatively complete inference code for ATST Frame (Audioset strong label)? Thanks for your help again!

**Comment by lmaxwell** [comment]
I write a quick solution in a new  pull request , can you test it ?

**Comment by Angelalilyer** [comment]
> I write a quick solution in a new pull request #13 , can you test it ?

thanks!!
I can run this code, but there are some labels in the "Labels" list that I cannot find their corresponding names in the "ontology.json" of strongly-labeled audioset. For example, "/m/0c1tlg"

**Comment by SaoYear** [comment]
The strong AudioSet includes some extra Mids excluded by the original AudioSet ontology, you could refer to the [official page](https://research.google.com/audioset/download_strong.html) and download the [mid_to_display_name.tsv](http://storage.googleapis.com/us_audioset/youtube_corpus/strong/mid_to_display_name.tsv) file. 

According the the file, `/m/0c1tlg` refers to `Electric rotor drone, quadcopter`.

**Comment by Angelalilyer** [comment]
> The strong AudioSet includes some extra Mids excluded by the original AudioSet ontology, you could refer to the [official page](https://research.google.com/audioset/download_strong.html) and download the [mid_to_display_name.tsv](http://storage.googleapis.com/us_audioset/youtube_corpus/strong/mid_to_display_name.tsv) file.
> 
> According the the file, `/m/0c1tlg` refers to `Electric rotor drone, quadcopter`.

Thank you very much for your help! The problem has been solved～

**Comment by folkartist** [comment]
> Hi, Currently, only self-supervised pretrained checkpoint are pulic in this repo. But we are happy to make the finetuned checkpoint public. As you are targeting for detercting sound events, do you mean the checkpoint and inference code of ATST-Frame finetuned on strongly-labeled audioset?
Where to download the checkpoint  file,  I can't find it  in readme file ..Tks
